universal-application-tool-0.0.1/app/services/role/RoleService.java

```diff
@@ -31,8 +32,7 @@ public RoleService(ProgramService programRepository, UserRepository userReposito
    * @return an {@link ImmutableSet} of {@link Account}s that are UAT admins.
    */
   public ImmutableSet<Account> getUatAdmins() {
```

Comment:
suggestion (non-blocking): I forgot about the rename - should this be `getCiviFormAdmins()`?

Reply:
`getGlobalAdmins` I think!